.gitignore

```diff
@@ -0,0 +1,7 @@
+file1?.txt
+file2?.txt
```

Comment:
Why is the rule for file2 here? Doesn't it state that the rule should not be there?

Reply:
This rule is not targetting file2.txt